frontend/nexus/post.ts

```diff
@@ -264,6 +267,10 @@ schema.extendType({
           data = { ...data, ...processEditorDocument(args.body) }
         }
 
+        if (args.status === 'PUBLISHED' && !originalPost.publishedAt) {
+          data = { ...data, publishedAt: new Date().toISOString() }
```

Comment:
```suggestion
          data.publishedAt = new Date().toISOString()
```

Doesn't affect anything other than not having to copy the whole `data` object